test/e2e/framework/fixture.go

```diff
@@ -258,19 +263,84 @@ func NewWorkspaceWithWorkloads(t *testing.T, server RunningServer, orgClusterNam
 	return orgClusterName.Join(ws.Name)
 }
 
-// StartWorkspaceSyncer starts a new syncer to synchronize the identified set of
-// resource types scheduled for the given workload cluster from the upstream server
-// to the downstream server.
-func StartWorkspaceSyncer(
+// SyncerFixture contains the information to run a syncer fixture.
+type SyncerFixture struct {
+	RunningServer       RunningServer
+	KubeClient          kubernetes.Interface
+	upstreamConfig      *rest.Config
+	downstreamConfig    *rest.Config
+	resources           sets.String
+	logicalClusterName  logicalcluster.LogicalCluster
+	WorkloadClusterName string
+	wsClusterName       logicalcluster.LogicalCluster
```

Comment:
Maybe s/wsClusterName/workspaceClusterName/ ? wsClusterName made sense in the context of the tests, but here being more verbose is probably suggested.